src/internal/connector/onedrive/drive.go

```diff
@@ -6,12 +6,11 @@ import (
 	"strings"
 
 	msgraphgocore "github.com/microsoftgraph/msgraph-sdk-go-core"
-	"github.com/microsoftgraph/msgraph-sdk-go/drives/item/items"
-	"github.com/microsoftgraph/msgraph-sdk-go/drives/item/items/item"
-	"github.com/microsoftgraph/msgraph-sdk-go/drives/item/root/delta"
+	"github.com/microsoftgraph/msgraph-sdk-go/drive"
+	msdrive "github.com/microsoftgraph/msgraph-sdk-go/drives"
 	"github.com/microsoftgraph/msgraph-sdk-go/models"
 	"github.com/microsoftgraph/msgraph-sdk-go/models/odataerrors"
-	mssitedrives "github.com/microsoftgraph/msgraph-sdk-go/sites/item/drives"
+	mssite "github.com/microsoftgraph/msgraph-sdk-go/sites"
```

Comment:
Similarly, since this is no longer a conflicting `drives` package, we could drop the alias for the more plain `sites`.
```suggestion
	"github.com/microsoftgraph/msgraph-sdk-go/sites"
```

Reply:
Resolved [a5ddaa3](https://github.com/alcionai/corso/pull/1698/commits/a5ddaa3e5c8f1d7b1ba1e017b896b0727dc0604a)